build.sh

```diff
@@ -32,5 +32,16 @@ if [[ -n "${SUFFIX}" ]]; then
 	SUFFIX="-${SUFFIX}"
 fi
 
-docker build --build-arg ARCH="${ARCH}" --build-arg MICROARCH="${MICROARCH}" --build-arg BOOTSTRAP="${BOOTSTRAP}" --build-arg SUFFIX="${SUFFIX}"  -t "${ORG}/${TARGET}:${VERSION}" -f "${NAME}.Dockerfile" .
+# Load local mirror as DIST
+DISTBUILDARG=''
+if [[ -e /etc/portage/make.conf ]]; then
+	export DIST=`grep 'GENTOO_MIRRORS="' /etc/portage/make.conf|cut -d '"' -f 2|cut -d ' ' -f1`
+	if [[ ${DIST} != "" ]]; then
```

Comment:
The `if [ "${DIST}" ]; then` would look a bit better and work the same.

Reply:
Agreed! Bash style isn't my forte...